packages/yoga/src/Button/web/Button.jsx

```diff
@@ -30,6 +31,8 @@ const Button = forwardRef(
       <StyledButton
         ref={ref}
         disabled={disabled}
+        aria-disabled={disabled || undefined}
```

Comment:
Isn't this enough? I mean, `false ?? undefined` will still be `false`
```suggestion
        aria-disabled={disabled}
```

Reply:
I did a test to see what the difference would be when `disabled` is `false` or `undefined`

With: `aria-disabled={disabled || undefined}`
![Screenshot 2023-09-26 at 10 50 51](https://github.com/gympass/yoga/assets/32910717/a78849e7-c745-4e59-9cf5-8a1fadedf786)

And with: `aria-disabled={disabled}`
![Screenshot 2023-09-26 at 10 51 28](https://github.com/gympass/yoga/assets/32910717/9f6e1873-5588-4655-939e-4754e0eeef2d)

Using `aria-disabled={disabled}` the aria-disabled attribute receives false, on the other hand, using `aria-disabled={disabled || undefined}` the aria-disabled attribute does not even appear.

I'm thinking what the best way would be, what do you think @MicaelRodrigues?

Reply:
@MicaelRodrigues I talked to the team and we realized that it would be good to keep `aria-disabled={disabled || undefined}.` 

It's more common the `aria-disabled `not to appear when it's `false`

Reply:
> I did a test to see what the difference would be when `disabled` is `false` or `undefined`
> 
> With: `aria-disabled={disabled || undefined}` ![Screenshot 2023-09-26 at 10 50 51](https://user-images.githubusercontent.com/32910717/270679895-a78849e7-c745-4e59-9cf5-8a1fadedf786.png)
> 
> And with: `aria-disabled={disabled}` ![Screenshot 2023-09-26 at 10 51 28](https://user-images.githubusercontent.com/32910717/270680064-9f6e1873-5588-4655-939e-4754e0eeef2d.png)
> 
> Using `aria-disabled={disabled}` the aria-disabled attribute receives false, on the other hand, using `aria-disabled={disabled || undefined}` the aria-disabled attribute does not even appear.
> 
> I'm thinking what the best way would be, what do you think @MicaelRodrigues?

> @MicaelRodrigues I talked to the team and we realized that it would be good to keep `aria-disabled={disabled || undefined}.`
> 
> It's more common the `aria-disabled `not to appear when it's `false`

@matheushdsbr  I understood all that... ;)
but if we want the default to be `undefined` we should just update the default prop value for `disabled` to `undefined`.

Take a look at the chat bellow. There are slight differences between `false` and `undefined` use cases. A `Button` user may want to explicitly set `aria-disabled` to `false` and with the current implementation he can't.

https://chat.openai.com/c/12783a4b-bd20-4823-bc2c-62b4fdb4f2f7

WDYT? :)

Reply:
Thanks for the tips, let's continue with changing `disabled` in `defaultProps` to `undefined`